README.md

```diff
@@ -21,10 +21,30 @@ React-Chatbot is a UI widget for adding [Vectara](https://vectara.com/)-powered
 
 ## UI
 
-_Screenshots coming soon!_
+React-Chatbot adds a button to the lower right corner of your application:
+
+<img alt="Chatbot toggle button" src="images/toggleButton.png" width=300></img>
+
+When the button is clicked, a chat window opens, where your users can start a conversation with your data:
+
+<img alt="Initial chat window state" src="images/emptyMessagesState.png" width=350></img>
+
+The Vectara platform responds to user messages and presents a list of references that support its answer:
+
+<img alt="Chat message answer" src="images/answerWithReferences.png" width=350></img>
+
+When the chat API returns an error, the UI allows the user to retry sending the message:
+
+<img alt="Retry message" src="images/messageWithError.png" width=350></img>
+
+Vectara's chatbot is also ready for use on mobile sites, taking the fullscreen when the chat window is opened:
```

Comment:
For consistency, I think we should change "Vectara's chatbot" to "React-Chatbot".

Spelling: "fullscreen" -> "full screen"

Reply:
And this is why I need you for reviews. 😆 "Vectara's chatbot" is absolutely not ideal.

Reply:
https://github.com/vectara/react-chatbot/pull/13/commits/908d9f7b3f5ab2e755606cd271302ce1306b7c1e